manual/manual/refman/exten.etex

```diff
@@ -620,16 +620,16 @@ select at run-time among several implementations of a signature.
 Each implementation is a structure that we can encapsulate as a
 first-class module, then store in a data structure such as a hash
 table:
-\begin{verbatim}
-        module type DEVICE = sig ... end
+\begin{caml_example*}{verbatim}
+        module type DEVICE = sig [@@@ellipsis.start][@@@ellipsis.stop] end
```

Comment:
Couldn't just `[@@@ellipsis]` work here?

Reply:
Yes, I was too focused on having tests for `[@ellipsis.start]...[@ellipsis.end]`.